source/tryCatch.js

```diff
@@ -22,8 +21,9 @@ import _curry2 from './internal/_curry2';
  * @example
  *
  *      R.tryCatch(R.prop('x'), R.F)({x: true}); //=> true
- *      R.tryCatch(JSON.parse, R.F)({});      //=> false
- */
+ *      R.tryCatch(R.test(/^a/, R.__), R.F)('b'); // => false
+ *      R.tryCatch(R.times(R.identity), R.always([]))('s') // => []
```

Comment:
I think perhaps we should add a much more explicit example:

   R.tryCatch(() => {throw 'oops'}, () => 'foo')('bar')  //=> 'foo'